### PR TITLE
Logging to stdout and better errors

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Added`
 
 - [#180](https://github.com/nf-core/epitopeprediction/pull/180) - Add support for `VEP` annotated VCF files [#172](https://github.com/nf-core/epitopeprediction/issues/172)
+- [#186](https://github.com/nf-core/epitopeprediction/pull/180) - Logging of epaa.py script also to stdout and added sys.exit error messages.
```
Comment (christopher-mohr) [inline CHANGELOG.md:11]:
```suggestion
- [#186](https://github.com/nf-core/epitopeprediction/pull/186) - Log messages from `epaa.py` script to stdout and provide `sys.exit` error messages.
```

File: bin/check_samplesheet.py [hunk 2]
```diff
@@ -325,7 +325,7 @@ def main(argv=None):
     logging.basicConfig(level=args.log_level, format="[%(levelname)s] %(message)s")
     if not args.file_in.is_file():
         logger.error(f"The given input file {args.file_in} was not found!")
-        sys.exit(2)
+        sys.exit(f"The given input file {args.file_in} was not found!")
```
Comment (christopher-mohr) [inline bin/check_samplesheet.py:328]: Not sure if we should change this since this is coming from the template.

File: bin/check_samplesheet.py [hunk 3]
```diff
@@ -199,7 +199,7 @@ def print_error(error, context="Line", context_str=""):
             error, context.strip(), context_str.strip()
         )
     print(error_str)
-    sys.exit(1)
+    sys.exit()
```
Comment (christopher-mohr) [inline bin/check_samplesheet.py:202]: Do we want exit code zero here?

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
👍 Some minor comments.

File: bin/check_samplesheet.py (hunk 2 above)
Comment (ggabernet) [reply bin/check_samplesheet.py]: you're right, it seems like some of those are fixed in the next template update

**Review by christopher-mohr — APPROVED** [review]
LGTM!

**Pull request merged**